Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class GameManager : MonoBehaviour {
+
+    [SerializeField]
+    int playerLives = 2;
```

Comment:
Always a good idea to throw the `private` before for clarity. so `private int playerLives = 2;` for example. Not declaring automatically makes it private but its good to put there anyways.